src/main/java/com/gregtechceu/gtceu/data/recipe/builder/GTRecipeBuilder.java

```diff
@@ -956,6 +953,14 @@ public GTRecipeBuilder outputFluids(FluidIngredient... outputs) {
         return output(FluidRecipeCapability.CAP, outputs);
     }
 
+    public GTRecipeBuilder outputFluidsRanged(Fluid output, IntProvider intProvider) {
+        return outputFluidsRanged(FluidIngredient.of(intProvider.getMaxValue(), output), intProvider);
```

Comment:
this should accept a `FluidStack` instead of just a `Fluid`, so:
```suggestion
    public GTRecipeBuilder outputFluidsRanged(FluidStack output, IntProvider intProvider) {
        return outputFluidsRanged(FluidIngredient.of(output), intProvider);
```